packages/conan/settings/profiles/vfx2024

```diff
@@ -3,5 +3,13 @@ include(ci_common4)
 [settings]
 python=3.11
 [options]
+# These options affects Qt only.
+qt:qtwebengine=True
+qt:gui=True
+qt:qtdeclarative=True
+qt:qtwebchannel=True
+qt:with_dbus=True
+qt:qtwebsockets=True
+qt:qtmultimedia=True
```

Comment:
I was following the "Conan" way of enabling options, but since we are already modifying the Qt recipe, I could set the defaults to `true` within the recipe directly (they are all set to `false` by default).